### ISSUE TITLE
Finding input images that cause high loss

### ISSUE THREAD

**Issue by taehyunzzz** [AlexeyAB/darknet#2873]
Hi, I'm trying to train YOLO with a custom dataset of mine, and I'm facing trouble with some of the training data. 

For some time, my  training seemed to be fine, but the avg loss stopped falling at 0.09. I noticed from my terminal during training phase that loss for certain batches were spiking, making the average loss 'un-fallable'. So I wanted to find out which input data is making the loss spike. 

So far, I've been modifying my yolov3 code in "train_network(~)" function in src/network.c and tried to write the input data that creates loss over 0.1 as jpg image while training my network on batch=2, subdivision=2. However, I could't quite get it done. Ultimately, I want to make a list of all input images as a .txt file so that I can easily check the input images creating this issue. 

I really think solving this issue would help all YOLOers find problems in their own dataset.
Any suggestions for how to modify my code?

Thanks in advance.

-Taehyun
